temporal-test-server/src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -2974,6 +2974,13 @@ private DescribeWorkflowExecutionResponse describeWorkflowExecutionInsideLock()
                 .setParentNamespaceId(p.getExecutionId().getNamespace())
                 .setParentExecution(p.getExecutionId().getExecution()));
 
+    if (this.startRequest.getCompletionCallbacksCount() > 0) {}
```

Comment:
Is this a typo or is github rendering this wrong?

Reply:
typo